conf/messages.en

```diff
@@ -99,6 +99,10 @@ signedOut.title = For your security, we signed you out
 signedOut.heading = For your security, we signed you out
 signedOut.guidance = We did not save your answers.
 
+notRegistered.title = You need to register for the Economic Crime Levy
+notRegistered.heading = You need to register for the Economic Crime Levy
+notRegistered.body = You need to <a href="https://www.gov.uk/anti-money-laundering-registration" class="govuk-link">register for the Economic Crime Levy (ECL)</a> before you can submit a return.
```

Comment:
this link is not the correct link 

it should be the start page of the registration journey for now until there is a GOV.UK guidance page in place for ECL registration